utility/src/main/java/org/oppia/android/util/platformparameter/PlatformParameterConstants.kt

```diff
@@ -92,3 +92,75 @@ const val CACHE_LATEX_RENDERING = "cache_latex_rendering"
 
 /** Default value for whether to cache LaTeX rendering using Glide. */
 const val CACHE_LATEX_RENDERING_DEFAULT_VALUE = true
+
+/** Qualifier for the platform parameter that controls whether to record performance metrics. */
+@Qualifier
+annotation class EnablePerformanceMetricsCollection
+
+/** Name of the platform parameter that controls whether to record performance metrics. */
+const val ENABLE_PERFORMANCE_METRICS_COLLECTION = "enable_performance_metrics_collection"
+
+/** Default value for whether to record performance metrics. */
+const val ENABLE_PERFORMANCE_METRICS_COLLECTION_DEFAULT_VALUE = false
+
+/**
+ * Qualifier for the platform parameter that controls the time interval in minutes of uploading
+ * previously recorded performance metrics to the remote service.
+ */
+@Qualifier
+annotation class PerformanceMetricsCollectionUploadTimeIntervalInMinutes
+
+/**
+ * Name of the platform parameter that controls the time interval in minutes of uploading previously
+ * recorded performance metrics to the remote service.
+ */
+const val PERFORMANCE_METRICS_COLLECTION_UPLOAD_TIME_INTERVAL_IN_MINUTES =
+  "performance_metrics_collection_upload_time_interval_in_minutes"
+
+/**
+ * Default value of the time interval in minutes of uploading previously recorded performance
+ * metrics to the remote service.
+ */
+const val PERFORMANCE_METRICS_COLLECTION_UPLOAD_TIME_INTERVAL_IN_MINUTES_DEFAULT_VAL = 15
+
+/**
+ * Qualifier for the platform parameter that controls the time interval in minutes of recording
+ * performance metrics that are to be recorded more frequently.
+ */
+@Qualifier
+annotation class PerformanceMetricsCollectionMoreFrequentRecordTimeIntervalInMinutes
+
+/**
+ * Name of the platform parameter that controls the time interval in minutes of recording
+ * performance metrics that are to be recorded more frequently.
+ */
+const val PERFORMANCE_METRICS_COLLECTION_MORE_FREQUENT_RECORD_TIME_INTERVAL_IN_MINUTES =
```

Comment:
I think we should be explicit about 'high frequency' vs 'low frequency'. There's an actual semantic difference here: 'high frequency' is describing the type of event being recorded, whereas 'more frequent record' is ambiguous and can be interpreted to mean "the time to record more frequently" (which isn't necessarily tied to a category of events).

I suggest renaming this & below, plus the corresponding module functions to be clearer.

Reply:
done.

Reply:
Sorry to clarify: I think we should actually name these 'high frequency' and 'low frequency', respectively, to make it sufficiently clear that these correspond to frequency vs. being frequent.

Reply:
done.